src/Config.ts

```diff
@@ -0,0 +1,119 @@
+import {Account} from "./Account";
+import {MosaicConfig} from "./MosaicConfig";
+import * as path from "path";
+import * as fs from 'fs-extra';
+
+// Database password key to read from env.
+const DB_PASSWORD: string = "MOSAIC_FACILITATOR_DB_PASSWORD";
+
+/**
+ * Holds mosaic config, database config and facilitator config.
+ */
+export class Config {
+    public database: Database;
+    public chains: Record<string, any>;
+    public mosaic: MosaicConfig;
+
+    /**
+     * Constructor.
+     * @param mosaicConfigPath Mosaic config path.
+     * @param facilitatorConfigPath Facilitator config path.
+     */
+    public constructor(
+        mosaicConfigPath: string,
+        facilitatorConfigPath: string
+    ) {
+        this.mosaic = MosaicConfig.fromPath(mosaicConfigPath);
+
+        const facilitatorConfig = this.getConfigJsonFromPath(
+            facilitatorConfigPath
+        );
+
+        this.database = new Database(facilitatorConfig);
+        this.chains = this.getChains(facilitatorConfig);
+    }
+
+    /**
+     * Get chain objects from given facilitator config.
+     * @param facilitatorConfig Facilitator config json data.
+     * @returns Array containing chain object.
+     */
+    private getChains(facilitatorConfig: Record<string, any>): Chain[] {
+        let chainsJson:Record<string, any> = facilitatorConfig.chains;
+        let chains:Chain[] = [];
+        for (let key in chainsJson) {
+            chains.push(new Chain(key, facilitatorConfig));
+        }
+        return chains;
+    }
+
+    private getConfigJsonFromPath(filePath: string): Record<string,any> {
+        if (fs.existsSync(filePath)) {
+            const config = fs.readFileSync(filePath).toString();
+            if (config && config.length > 0) {
+                return JSON.parse(config);
+            }
+        }
+        return null;
+    }
+}
+
+/**
+ * Holds database configurations.
+ */
+export class Database {
```

Comment:
I feel we should call the class DBConfig as it only holds the config to make a db connection.